odkx-src/xlsx-converter-reference.rst

```diff
@@ -482,6 +482,16 @@ Available :th:`setting_name` values that can be used:
     - | Optional
     - | Used with :th:`display.prompt.text.<language>`, or
       | other fiels to set other language options in the form.
+  * - showHeader
+    - | Optional
+    - | Used to display the header at the top of every page of the form.
+      | Shows the form title in a navigation bar with Back and Next buttons.
```

Comment:
Where the form title is has a button for the menu, so maybe here "Shows the menu with the form title in a navigation bar with Back and Next buttons."

Reply:
noted! thank you for the suggesition